packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -440,7 +442,15 @@ function _finalizeElementPosition(
   const hostRect: Rectangle = _getRectangleFromElement(hostElement);
   const elementEdge = coverTarget ? targetEdge : targetEdge * -1;
   const elementEdgeString = RectangleEdge[elementEdge];
-  const returnEdge = alignmentEdge ? alignmentEdge : _getFlankingEdges(targetEdge).positiveEdge;
+  let returnEdge = alignmentEdge ? alignmentEdge : _getFlankingEdges(targetEdge).positiveEdge;
+
+  // if the element is closer to one side of the bounds than the other, flip the return edge to ensure it grows inwards
+  if (bounds) {
+    const flankingEdges = _getFlankingEdges(targetEdge);
```

Comment:
On second look, this could be a bit cleaner if you just get the opposite of the alignment edge, that way you don't need to worry about calling getFlankingEdges again. Not necessary, just a minor suggestion.